test/inductor/test_torchinductor.py

```diff
@@ -93,6 +94,8 @@
 
 DO_PERF_TEST = os.environ.get("DO_PERF_TEST") == "1"
 
+ROCM_WHEELS_ENV = TEST_WITH_ROCM and not HAS_HIPCC
```

Comment:
@iupaikov-amd Do we need `TEST_WITH_ROCM` check as well? Is the HAS_HIPCC check not enough, since that includes the `torch.version.hip is not None` to ensure it's a ROCm build?

Reply:
I followed the logic that I found with similar skips for different tests, they tend to check this variable as well. I can remove it if it feels redundant.

Reply:
I think this can help us avoid skipping tests for nvidia systems that have parts of rocm installed but don't use it. Just a safety precaution for upstream.

Reply:
"nvidia systems that have parts of rocm installed" That's a null set afaik. :) But your main point is valid that we should *not* skip this test for CUDA environment, only for ROCm environment if it doesn't have hipcc.

Reply:
As discussed offline though, I'm okay with the `TEST_WITH_ROCM and not HAS_HIPCC` condition for release branches. I'd just like to make it even more concise for the upstream version.

Reply:
> And to the above point, I think you hedged your bets with the `torch.version.hip is not None and ROCM_HOME is not None` in `HAS_HIPCC` definition, so I'm still trying to reason why additional `TEST_WITH_ROCM` condition is needed.

Sorry, I do see now why `TEST_WITH_ROCM` is needed to make the CUDA env not skip these unit tests. We'll explore how to improve this and other ROCm-specific skip logic in a future PR.